CPAN/buildme.sh

```diff
@@ -463,7 +463,7 @@ elif [ "$OS" = "SunOS" ]; then
         echo "ERROR: Please install GNU make (gmake)"
         exit
     fi 
-    export MAKE=/usr/bin/gmake
+    export MAKE="/usr/bin/gmake -j4"
```

Comment:
Does j4 make sense here. When one simple wants to build on 1 core or a different number...

Reply:
It only makes sense if the cpu is quad core (Or dual w/threads). Do you have a build problem if you omit the -j?